applications/tasks/src/kz_task_worker.erl

```diff
@@ -223,7 +225,7 @@ is_task_successful(MappedRow
                     {'false', Columns, Written, NewIterValue}
             end;
         Fields ->
-            PPFields = kz_binary:join(Fields, $\s),
+            PPFields = kz_binary:join(Fields, <<$\s>>),
```

Comment:
It was either update the spec for join or change the arg here. Wasted memory = 6 words?